src/jdk.jpackage/windows/classes/jdk/jpackage/internal/resources/WinResources.properties

```diff
@@ -60,7 +60,7 @@ message.output-location=Installer (.exe) saved to: {0}
 message.tool-version=Detected [{0}] version [{1}].
 message.creating-association-with-null-extension=Creating association with null extension.
 message.wrong-tool-version=Detected [{0}] version {1} but version {2} is required.
-message.version-string-too-many-components=Version sting may have up to 3 components - major.minor.build .
+message.version-string-too-many-components=Version string may have up to 3 components - major.minor.build .
```

Comment:
I wonder whether the space before the period is required at the end of the sentence. Perhaps, it's to separate a property name from the end of the sentence.

Reply:
right; without the space the period would appear to be part of the version pattern.

Reply:
Yes, I believe this is why it was done.